Firmware/temp_model.h

```diff
@@ -59,6 +59,7 @@ struct model_data
 };
 
 static bool enabled;          // model check enabled
+static bool valid = false;    // model is valid
```

Comment:
I'm not sure why this is needed, can you explain?
temp_model::calibrated() will return true if the data is ok, so we shouldn't need this extra variable?